components/sideNav.tsx

```diff
@@ -11,7 +12,14 @@ type props = {
 const SideNav=({userData}:props)=>{
     let [state,open]=React.useState(!(Object.getPrototypeOf(userData) === Object.prototype))
     function menu(){
-        open(state=!state)
+        // did not understand how to implement if/else here :( => (note by Kris)
+        // open(state=!state)
+        if(Object.getPrototypeOf(userData) == null) {
+            open(!state)
+        }
+        else {
+            open(state=!state)
+        }
```

Comment:
I handled this already

Reply:
so return it to the original, instead of disabling a user to open it, they will open it but see different data based on whether they have clicked on a maker or not